projects/kserve/visualizations/kserve-llm/models/kpi.py

```diff
@@ -51,6 +51,43 @@ def kserve_llm_load_test_tpot_p95(lts_payload):
 def kserve_llm_load_test_tpot_p99(lts_payload):
     return lts_payload.results.time_per_output_token.percentile_99
 
+# Inter token latency
+
+@matbench_models.KPIMetadata(help="All values of inter token latency", unit="List[ms]")
+def kserve_llm_load_test_itl(lts_payload):
+    return lts_payload.results.inter_token_latency.values if hasattr(lts_payload.results, "inter_token_latency") else []
```

Comment:
can you test if `lts_payload.results.inter_token_latency` is None instead of  `hasattr`, here an below?
(see the other comment to always define it)